src/it/projects/sources/test.properties

```diff
@@ -16,3 +16,4 @@
 # under the License.
 
 outputFile = target/resolved.txt
+classifier = foo
```

Comment:
This may require a comment that it has no influence anymore...

Reply:
Will do

Reply:
this test can be useful ... even we have a read only property user still can override it

Reply:
and there are no assertions in this test

Reply:
I guess at least the build should not fail, but need to try...